### PR TITLE
Get-AzEffectiveRouteTable powershell cmdlet not showing disableBgpRoutePropagation property

### PR DESCRIPTION
**Pull request by wdehrich**
## Checklist

- [ ] I have read the [_Submitting Changes_](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md)
- [ ] The title of the PR is clear and informative
- [ ] The appropriate [change log has been updated](https://github.com/Azure/azure-powershell/blob/master/CONTRIBUTING.md#updating-the-change-log)
- [ ] The PR does not introduce [breaking changes](https://github.com/Azure/azure-powershell/blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [ ] If applicable, the changes made in the PR have proper test coverage
- [ ] For public API changes to cmdlets:
    - [ ] the changes have gone through a [cmdlet design review](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (Microsoft internal partners only)
    - [ ] the cmdlet markdown files were [generated using the `platyPS` module](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/help-generation.md)

### CODE REVIEW

**Comment by markcowl** [comment]
@wdehrich Please resolve merge conflicts

**Comment by markcowl** [comment]
@wadeheninger Please update this PR to fix merge conflicts.  I will have to close this PR at EOW for lack of activity, otherwise

**Comment by wdehrich** [comment]
@markcowl - I made changes was to the CSPROJ, NUGET, and CONFIG files to incorporate an update to the API/SDK that has since been merged: [https://github.com/Azure/azure-rest-api-specs/pull/5585/files](https://github.com/Azure/azure-rest-api-specs/pull/5585/files)

Should I remove the changes to the CSPROJ, NUGET, and CONFIG files in the merge?

File: src/Network/Network/Models/PSEffectiveRoute.cs [hunk 1]
```diff
@@ -25,6 +25,10 @@ public class PSEffectiveRoute
         [Ps1Xml(Target = ViewControl.Table)]
         public string Name { get; set; }
 
+        [JsonProperty(Order = 1)]
```
Comment (markcowl) [inline src/Network/Network/Models/PSEffectiveRoute.cs]: How is this going to be populated?  Does it need an update to the Map for this type?

Comment (markcowl) [inline src/Network/Network/Models/PSEffectiveRoute.cs]: Need a changelog entry

Comment (wdehrich) [reply src/Network/Network/Models/PSEffectiveRoute.cs]: No update to map needed

Comment (wdehrich) [reply src/Network/Network/Models/PSEffectiveRoute.cs]: Added a changelog entry.

**Pull request merged**